src/main/java/org/jenkinsci/plugins/workflow/cps/CpsBodyExecution.java

```diff
@@ -119,6 +119,8 @@
                 sn.addAction(a);
         }
 
+        CpsFlowExecution.maybeAutoPersistNode(sn);  // Hacky but helpful
```

Comment:
But not the most helpful comment. :-) Is there something a reader should know? I for one have no idea what this is about.